python/ray/autoscaler/updater.py

```diff
@@ -423,16 +423,16 @@ def do_update(self):
             # Run init commands
             self.provider.set_node_tags(
                 self.node_id, {TAG_RAY_NODE_STATUS: STATUS_SETTING_UP})
-            with LogTimer(self.log_prefix +
-                          "Initialization commands completed"):
+            with LogTimer(self.log_prefix + "Initialization commands",
+                          "completed"):
                 for cmd in self.initialization_commands:
                     self.cmd_runner.run(cmd)
 
-            with LogTimer(self.log_prefix + "Setup commands completed"):
+            with LogTimer(self.log_prefix + "Setup commands", "completed"):
```

Comment:
any reason to have an explicit success str?

Reply:
Not really--are you suggesting that all commands have "success" upon completion and "failed" upon failure?

Reply:
yeah, maybe flag it off with `LogTimer(msg, show_status=True)`

Reply:
Good idea!